circle.yml

```diff
@@ -0,0 +1,8 @@
+machine:
+  node:
+    version: 5
+
+dependencies:
+  override:
+    - npm install
+    - ./node_modules/.bin/bower install
```

Comment:
Hmm, that is unfortunate. I would expect it to be there. Maybe even better would be to add a `postinstall` task that does this just like takeout-management does.